app/models/game.rb

```diff
@@ -1,3 +1,46 @@
 class Game < ActiveRecord::Base
-	belongs_to :name
+	belongs_to :type
+	after_create :populate_board!
+
+	def populate_board!
+		my_game = game.create
+		my_game.pieces.create(:x=>'a', :y=>'1',:type=>'rook', :color=>'white')
+		my_game.pieces.create(:x=>'b', :y=>'1', :type=>'knight', :color=>'white')
+		my_game.pieces.create(:x=>'c', :y=>'1', :type=>'bishop', :color=>'white')
+		my_game.pieces.create(:x=>'d', :y=>'1', :type=>'queen', :color=>'white')
+		my_game.pieces.create(:x=>'e', :y=>'1', :type=>'king', :color=>'white')
+		my_game.pieces.create(:x=>'f', :y=>'1', :type=>'bishop', :color=>'white')
+		my_game.pieces.create(:x=>'g', :y=>'1', :type=>'knight', :color=>'white')
+		my_game.pieces.create(:x=>'h', :y=>'1', :type=>'rook', :color=>'white')
+		my_game.pieces.create(:x=>'a', :y=>'2', :type=>'pawn', :color=>'white')
+		my_game.pieces.create(:x=>'b', :y=>'2', :type=>'pawn', :color=>'white')
+		my_game.pieces.create(:x=>'c', :y=>'2', :type=>'pawn', :color=>'white')
+		my_game.pieces.create(:x=>'d', :y=>'2', :type=>'pawn', :color=>'white')
+		my_game.pieces.create(:x=>'e', :y=>'2', :type=>'pawn', :color=>'white')
+		my_game.pieces.create(:x=>'f', :y=>'2', :type=>'pawn', :color=>'white')
+		my_game.pieces.create(:x=>'g', :y=>'2', :type=>'pawn', :color=>'white')
+		my_game.pieces.create(:x=>'h', :y=>'2', :type=>'pawn', :color=>'white')
+
+		my_game.pieces.create(:x=>'a', :y=>'8',:type=>'rook', :color=>'black')
+		my_game.pieces.create(:x=>'b', :y=>'8', :type=>'knight', :color=>'black')
+		my_game.pieces.create(:x=>'c', :y=>'8', :type=>'bishop', :color=>'black')
+		my_game.pieces.create(:x=>'d', :y=>'8', :type=>'queen', :color=>'black')
+		my_game.pieces.create(:x=>'e', :y=>'8', :type=>'king', :color=>'black')
+		my_game.pieces.create(:x=>'f', :y=>'8', :type=>'bishop', :color=>'black')
+		my_game.pieces.create(:x=>'g', :y=>'8', :type=>'knight', :color=>'black')
+		my_game.pieces.create(:x=>'h', :y=>'8', :type=>'rook', :color=>'black')
+		my_game.pieces.create(:x=>'a', :y=>'7', :type=>'pawn', :color=>'black')
+		my_game.pieces.create(:x=>'b', :y=>'7', :type=>'pawn', :color=>'black')
+		my_game.pieces.create(:x=>'c', :y=>'7', :type=>'pawn', :color=>'black')
+		my_game.pieces.create(:x=>'d', :y=>'7', :type=>'pawn', :color=>'black')
+		my_game.pieces.create(:x=>'e', :y=>'7', :type=>'pawn', :color=>'black')
+		my_game.pieces.create(:x=>'f', :y=>'7', :type=>'pawn', :color=>'black')
+		my_game.pieces.create(:x=>'g', :y=>'7', :type=>'pawn', :color=>'black')
+		my_game.pieces.create(:x=>'h', :y=>'7', :type=>'pawn', :color=>'black')
+
+		if my_game.pieces.count == 32
+			return true
+		end
```

Comment:
Looks great. :smile:. You won't need line 41-43.

`my_game.pieces.create` will return true on the last line. :)